Cognite.Simulator.Tests/UtilsTests/SimulationRunnerTest.cs

```diff
@@ -231,6 +250,160 @@ public async Task TestSimulationRunnerBase()
+
+                // Run model and configuration libraries to fetch the test model and
+                // test simulation configuration from CDF
+                await modelLib.Init(source.Token).ConfigureAwait(false);
+                await configLib.Init(source.Token).ConfigureAwait(false);
+
+                using var linkedTokenSource = CancellationTokenSource.CreateLinkedTokenSource(source.Token);
+                var linkedToken = linkedTokenSource.Token;
+                linkedTokenSource.CancelAfter(TimeSpan.FromSeconds(6));
+                var taskList = new List<Task>(modelLib.GetRunTasks(linkedToken));
+                taskList.AddRange(configLib.GetRunTasks(linkedToken));
+                await taskList.RunAll(linkedTokenSource).ConfigureAwait(false);
+
+                Assert.NotEmpty(configLib.State);
+                var configState = Assert.Contains(
+                    "PROSPER-SC-UserDefined-SRT-Connector_Test_Model", // This simulator configuration should exist in CDF
+                    (IReadOnlyDictionary<string, TestConfigurationState>)configLib.State);
+                var configObj = configLib.GetSimulationConfiguration(
+                    "PROSPER", "Connector Test Model", "UserDefined", "SRT");
+                Assert.NotNull(configObj);
+
+                var outTsIds = configObj.OutputTimeSeries.Select(o => o.ExternalId).ToList();
+                tsToDelete.AddRange(outTsIds);
+                var inTsIds = configObj.InputTimeSeries.Select(o => o.SampleExternalId).ToList();
+                tsToDelete.AddRange(inTsIds);
+
+                // Create a simulation event ready to run for the test configuration
+                var simRuns = await cdf.Alpha.Simulators.CreateSimulationRunsAsync(
+                    new List<SimulationRunCreate>
+                    {
+                        new SimulationRunCreate
+                        {
+                            ModelName = configObj.ModelName,
+                            RoutineName = configObj.CalculationName,
+                            SimulatorName = configObj.Simulator
+                        }
+                    }, source.Token).ConfigureAwait(false);
+                Assert.NotEmpty(simRuns);
+                runId = simRuns.First().Id;
+
+                // Run the simulation runner and verify that the event above was picked up for execution
+                using var linkedTokenSource2 = CancellationTokenSource.CreateLinkedTokenSource(source.Token);
+                var linkedToken2 = linkedTokenSource2.Token;
+                linkedTokenSource2.CancelAfter(TimeSpan.FromSeconds(15));
+                var taskList2 = new List<Task> { runner.Run(linkedToken2) };
+                await taskList2.RunAll(linkedTokenSource2).ConfigureAwait(false);
+
+                Assert.True(runner.MetadataInitialized);
+
+                // Uncomment when we have runTime param in place
```

Comment:
not runTime but validatioEndTime

Reply:
change aggregation type for stepInterpolation and in that case the test will succeed regardless of wether we have dps or not